src/wizard/wizard.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <v-container>
+      <v-row>
+        <v-col cols="12">
+          <Stepper />
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12">
+          <Step1 />
```

Comment:
you might want to change this for 

` <router-view></router-view>`

when using nesting routes:

https://router.vuejs.org/guide/essentials/nested-routes.html

Reply:
ok, will review and assess this and push any changes up later. Thx for the information. For now, you and Devante have what you need to get started. Please approve and merge.